spring-web/src/main/java/org/springframework/http/client/OkHttp3ClientHttpRequestFactory.java

```diff
@@ -125,7 +125,7 @@ static Request buildRequest(HttpHeaders headers, byte[] content, URI uri, HttpMe
 		okhttp3.MediaType contentType = getContentType(headers);
 		RequestBody body = (content.length > 0 ||
 				okhttp3.internal.http.HttpMethod.requiresRequestBody(method.name()) ?
-				RequestBody.create(contentType, content) : null);
+				RequestBody.create(content, contentType) : null);
```

Comment:
The old `RequestBody.create()` from OkHttp 3 which was used here is still present in OkHttp 4, but marked as `Deprecated`. This change was necessary because the build-process refuses to compile against deprecated methods.